SDF-to-Coords

```diff
@@ -0,0 +1,31 @@
+import os
+
+def SDF_to_Coords(filename, num_atom):
+    directory = r'/mnt/d/SDFs/structures/'
+    for filename in os.listdir(directory):
+        if filename.endswith(".sdf"):
+            with open(filename) as f:
+                xyz = []
```

Comment:
This function effectively produces the list we want. But there are many lines that follow the list, and the single entry on each output line is "[ ]". I haven't been able to resolve the issue. After this is resolved, I'll change xyz from a list to a matrix, and switch the last column to the first.

Reply:
I'll give you a hint, on line 17 try printing the state of the loop, *i.e.*, put something like:

```.py
print(line, counter, num_atom)
```